### DISCUSSION TITLE
clip-based workflow

### DISCUSSION THREAD

**Discussion by boyberg** [bangcorrupt/hacktribe#118] [Ideas]
do you think it would be possible to implement some kind of clip-functionality? i image something like cycling through multiple clips on one pad with part+shift and then arranging them into scenes in the utility menu. any thoughts on this?

**Comment by bangcorrupt** [comment]
I'd like to be able to select sequences from different patterns like on a lot of other grooveboxes.  Making the event recorder an editable linear sequencer would also be cool.

**Comment by redwallx140** [comment]
When I need more sequence space, I resample the overall parts and create a pattern at 1/4 the tempo. I use the steps slots as triggers for the  parts, and this gives me a quasi-clip grid. Then, step jump lets me cycle through the arrangement.  It's another way to fiddle with the concept of time in the device.  Maybe this is a way to reframe what clip mode is without re-inventing capability.

**Reply by listenForward** [reply]
I LOVE your idea of re-sampling parts and slicing them to 1/4 note beat-clips along the Grid...
It's given me the idea for [my own post of brainstorming](https://github.com/bangcorrupt/hacktribe/discussions/203) some approaches the get the Trigger Mode(s) closer to come "clip launching" paradigm.
